README.md

```diff
@@ -33,89 +33,92 @@ System.Web.Bem - БЭМ-инфрастурктура для ASP.NET MVC.
+
+Assembling is performed by special tool [enb](https://ru.bem.info/toolbox/enb), which is added to the project during the System.Web.Bem package installation from NuGet. Also during the package installation the project settings is changed: the automatical enb execution after the MsBuild execution is added. Thus, when you run the build of your ASP.NET MVC project in Visual Studio the assembling of BEM-bundles will be performed automatically after the c# code compilation. During the project assembling will be found all the declarations in the `/Bem/desktop.bundles` folder and for each of them will be assembled all the technologies bundles (templates, styles, java script).
+
+Bundles of technologies should be named `<bundle_name>.<tech_ext>` and should be saved in the folder where the declaration is located. E.g. the templates file (`bemhtml.js`) for the `default` bundle should have a path `/Bem/desktop.bundles/default/default.bemhtml.js`. Thus, after the build of the project you will see something like this:
 
-Бандлы технологий имеют имя `<bundle_name>.<tech_ext>` и сохраняются в папку, где находится декларация. Например, файл шаблонов (bemhtml.js) для бандла `default` будет иметь путь `/Bem/desktop.bundles/default/default.bemhtml.js`. Таким образом, после сбоки проекта вы будете наблюдать примерно такую картину:
 ```
 <Project root>
 ├─ Bem
 │  ├─ ...
 │  └─ desktop.bundles
 │     ├─ default 
-│     │  ├─ default.bemdecl.js  // декларация бандла
-│     │  ├─ default.bemhtml.js  // бандл с шаблонами 
-│     │  ├─ default.js          // бандл с клиентским кодом 
-│     │  └─ default.css         // бандл со стилями 
-│     │  └─ ...                 // бандлы других технологий
+│     │  ├─ default.bemdecl.js  // bundle declaration
+│     │  ├─ default.bemhtml.js  // templates bundle 
+│     │  ├─ default.js          // client-side code bundle 
+│     │  └─ default.css         // styles bundle 
+│     │  └─ ...                 // bundles of other technologies
 │     └─ ...
 └─ ...
 ```
-Во время работы приложения бандл шаблонов используется для формирования html (на сервере и клиенте), бандлы js и css подключаются на страницы и используются на клиенте. Во время серверной шаблонизации будет использоваться бандл с шаблонами, указанный в настройках приложения. Бандлы js и css нужно самостоятельно подключить на страницу.
 
-### Серверная шаблонизация BEMHTML
+During the application is running the bundle of templates is used for html generation (on the server and the client side), js bundles and style bundles are included to the pages and used on the client side. During the server-side html generation the templates bundle specified in application settings will be used. Client-side bundles (js and css) should be included to the page manually.
```

Comment:
-> When the application is running, the bundle of templates is used for generating HTML (on the server and the client side), while JS bundles and style bundles are included to the pages and used on the client side. Server-side HTML generation uses the bundle of templates specified in application settings. The client-side bundles (JS and CSS) should be included into the page manually.